lib/go-tc/users.go

```diff
@@ -107,20 +109,27 @@ type User struct {
 	commonUserFields
 }
 
-// UserV40 contains ChangeLogCount field
+// UserV40 contains ChangeLogCount field.
 type UserV40 struct {
 	User
-	ChangeLogCount *int `json:"changeLogCount" db:"change_log_count"`
+	ChangeLogCount    *int       `json:"changeLogCount" db:"change_log_count"`
+	LastAuthenticated *TimeNoMod `json:"lastAuthenticated" db:"last_authenticated"`
 }
 
-// UserCurrent represents the profile for the authenticated user
+// UserCurrent represents the profile for the authenticated user.
 type UserCurrent struct {
 	UserName  *string `json:"username"`
 	LocalUser *bool   `json:"localUser"`
 	RoleName  *string `json:"roleName"`
 	commonUserFields
 }
 
+// UserCurrentV40 contains LastAuthenticated field.
+type UserCurrentV40 struct {
+	UserCurrent
+	LastAuthenticated *TimeNoMod `json:"lastAuthenticated" db:"last_authenticated"`
```

Comment:
Can we just make this a `time.Time`? That way it comes out as RFC3339-formatted instead of our proprietary format.

Reply:
ok.